src/quake/q_packfile.c

```diff
@@ -121,10 +121,17 @@ i32 SearchPath_FindPack(SearchPath* sp, const char* path)
     return -1;
 }
 
+// not happy with how this looks tbh.
+#if PLAT_WIN
+#define PAK_GLOB "%s/*.pak"
+#else
+#define PAK_GLOB "%s/*.[pP][aA][kK]"
```

Comment:
This is the hacky current solution for case insensitivity that I would like to change.